app/experimenter/experiments/models/nimbus.py

```diff
@@ -268,6 +268,26 @@ def proposed_end_date(self):
                 self.start_date + datetime.timedelta(days=self.proposed_duration)
             ).date()
 
+    @property
+    def computed_enrollment_days(self):
+        paused_change = (
+            self.changes.all()
+            .filter(experiment_data__is_paused=True)
+            .order_by("changed_on")
+            .first()
+        )
+        if paused_change:
+            return (paused_change.changed_on - self.start_date).days
+        else:
+            return self.proposed_enrollment
+
+    @property
+    def computed_duration(self):
+        if self.start_date:
+            return (self.computed_end_date - self.start_date).days
+        else:
+            return self.proposed_duration
+
     @property
     def computed_end_date(self):
         return self.end_date or self.proposed_end_date
```

Comment:
@lmorchard I reverted this back to what we had before because it seems to be fine? Not sure what hackery we ran into yesterday but I checked a bunch of completed experiments locally and things look as expected to me.

Reply:
Ah, yeah, this is probably fine. A proper if/else might be clearer, but if it ain't broke don't poke it

Reply:
Oh yeah python short circuiting, it'll return the first truthy value.  I tend to use it for very simple cases like this, you can technically build complex logical expressions and get values out of it but that's usually better written as if/else, but for simple things like this I just like to use the short circuited version